commands/list_releases.go

```diff
@@ -64,32 +64,35 @@ func init() {
 	ListCommand.AddCommand(ListReleasesCommand)
 }
 
-// listReleasesValidationOutput does our pre-checks and shows errors, in case
+// listReleasesPreRunOutput does our pre-checks and shows errors, in case
 // something is missing.
-func listReleasesValidationOutput(cmd *cobra.Command, extraArgs []string) {
+func listReleasesPreRunOutput(cmd *cobra.Command, extraArgs []string) {
 	args := defaultListReleasesArguments()
-	err := listReleasesValidate(&args)
-	if err != nil {
-		handleCommonErrors(err)
+	err := listReleasesPreconditions(&args)
 
-		fmt.Println(color.RedString(err.Error()))
-		os.Exit(1)
+	if err == nil {
+		return
 	}
+
+	handleCommonErrors(err)
+
+	fmt.Println(color.RedString(err.Error()))
+	os.Exit(1)
```

Comment:
Line 73 to 80 are unrelated to the cause. It's alignment with our command blueprint.